docs/src/modules/components/Ad.js

```diff
@@ -34,7 +34,10 @@ function getAdblock(classes, t) {
         {t('likeMui')}
       </Typography>
       <Typography variant="body2" display="block" component="span" gutterBottom>
-        {t('adblock')}{' '}
+        {t('adblock')}
+      </Typography>
+      <Typography variant="body2" display="block" component="span" gutterBottom>
+        {t('thanks')}{' '}
         <span role="img" aria-label={t('emojiLove')}>
           ❤️
         </span>
```

Comment:
While on the topic of ads (can't add a comment further down), they aren't disabled during development.
https://github.com/mui-org/material-ui/blob/405b3c1404ed2f2c0c27378a7356b2a67562bfc4/docs/src/modules/components/Ad.js#L49-L50

Reply:
Oop, thanks 👍